Dockerfile

```diff
@@ -1,8 +1,22 @@
-FROM ubuntu:jammy 
+FROM datajoint/pydev:python3.6
+
+RUN apt update && apt -y install mysql-client-5.7 netcat
+
 ENV TEAM="Seahawks"
-COPY ./README.md /home/ubuntu/files/README.md
-ADD https://raw.githubusercontent.com/datajoint/datajoint-python/master/CHANGELOG.md /home/ubuntu/files/CHANGELOG.md
-RUN echo HEY! > /home/ubuntu/files/file1.txt
-USER games:games
+
+RUN mkdir /seahawks
+
+WORKDIR /seahawks
+
+RUN git clone https://github.com/davidgodinez/seahawks-newsletter-pipeline.git /seahawks
+
+RUN pip install --upgrade pip && pip install .
+
+RUN pip install -r requirements.txt
```

Comment:
What would happen if you were to remove this? What would be the implications of not having this included in your image?

Reply:
Removing `pip install -r requirements.txt` shouldn't have any negative consequences because this file will be run when `setup.py` is called.